static/js/place/place_template.tsx

```diff
@@ -783,12 +783,24 @@ class Chart extends Component<ChartPropType, ChartStateType> {
             } else if (this.placeRelation === placeRelationEnum.CONTAINING) {
               placesPromise = this.props.childPlacesPromise.then(
                 (childPlaces) => {
-                  // TODO(boxu): figure out a better way to pick child places.
+                  // Choose the place type that has the highest average
```

Comment:
How do we handle places with multiple types?

Reply:
The children places is generated in https://github.com/datacommonsorg/website/blob/master/server/routes/api/place.py#L157-L170

- A map from place type to list of places are first generated from containedInPlace relation 
- If there are equivalent place types, then the preferred one is chosen.